crates/core/src/dom/mod.rs

```diff
@@ -533,8 +533,8 @@ impl Document {
         if patches.is_empty() {
             return Ok(());
         }
-        let handler = self.event_callback.clone();
 
+        let handler = self.event_callback.clone();
```

Comment:
I fail to see how this line is changed.

Reply:
It isn't I just added newline I forgot to remove while I was messing around with some other patch inspection ideas. I'll revert it.